README.md

```diff
@@ -44,6 +61,22 @@ SCOUT_KEY = "[AVAILABLE IN THE SCOUT UI]"
 SCOUT_NAME = "A FRIENDLY NAME FOR YOUR APP"
 ```
 
+### Falcon
+
+```python
+import falcon
+from scout_apm.falcon import ScoutMiddleware
+
+scout_middleware = ScoutMiddleware(config={
+    "key": "[AVAILABLE IN THE SCOUT UI]",
+    "monitor": True,
+    "name": "A FRIENDLY NAME FOR YOUR APP",
+})
+api = falcon.API(middleware=[ScoutMiddleware()])
+# Required for accessing API internals, Falcon doesn't provide it default
```

Comment:
"by default"

Reply:
Right, this doesn't make sense to say - it doesn't provide it all.